scripts/setup-operators.sh

```diff
@@ -51,25 +51,11 @@ do
     fi
 done
 
-# Now onto namespace bound operator
-# Create OperatorGroup
-oc create -f - <<EOF
```

Comment:
You are removing this because now you are using the default operator namespace `openshift-operators`, right?

Reply:
Not "default" really. The `openshift-operators` namespce already has an Operator Group when the OpenShift cluster comes up. So, we don't need to create one ourselves.

Reply:
```sh
$ oc get og -n openshift-operators
NAME               AGE
global-operators   20d
```

Reply:
cool!!!